File: Programing C++/lab2/py/main.py
import func
def main():
    num = func.set_num_show()
    shows = func.set_shows(num)
    mode = func.choose_mode()
    func.create_file("first", shows, mode)
    shows = func.read_file("first")
    print("Shows:")
    func.print_file("first")
    print("Durations of shows:")
    func.print_show_duration(shows)
    new_shows = func.set_day_shows(shows)
    func.create_file("second", new_shows, "wb")
    new_shows = func.read_file("second")
    print("Day shows:")
    func.print_file("second")

if __name__ == '__main__':
    main()

File: Programing C++/lab2/cpp/main.cpp
#include "func.h"
int main()
{
	srand(time(NULL));
	int num = set_num_show();
	vector<TvShow> shows = set_shows(num);
	int mode = choose_mode();
	create_file("first", shows, mode);
	shows = read_file("first");
	cout << "Shows: " << endl;
	print_file("first");
	cout << "Durations of shows: " << endl;
	print_show_duration(shows);
	vector<TvShow> new_shows = set_day_shows(shows);
	create_file("second", new_shows, 1);
	new_shows = read_file("second");
	cout << "Day shows: " << endl;
	print_file("second");
	return 0;
}